Community Balance Overhaul/Art/Improvements/Inca/CBP_IncaFunctions.lua

```diff
@@ -57,8 +45,8 @@ function JFD_CBP_Inca_MachuPicchu(playerID, plotX, plotY)
 		end
 	end
 end
-if (userSettingInca and isIncaCivActive) then
+if isIncaCivActive then
 	GameEvents.PlayerCityFounded.Add(JFD_CBP_Inca_MachuPicchu)
```

Comment:
What is the point of this line? It looks like any city founded on a mountain gets the JFD_CBP_Inca_MachuPicchu Improvement, even if Inca is not in the game.

Reply:
I mean that, in the *actual game*, Civs that aren't Inca will get the JFD Machu Picchu improvement when they settle on mountains, even if Inca isn't in the game.